xero-payroll-au.yaml

```diff
@@ -3381,6 +3383,18 @@ components:
           $ref: '#/components/schemas/OpeningBalances'
         TaxDeclaration: 
           $ref: '#/components/schemas/TaxDeclaration'
+        IncomeType: 
+          $ref: '#/components/schemas/IncomeType'
+        EmploymentType: 
+          $ref: '#/components/schemas/EmploymentType'
+        CountryOfResidence:
+          description: Country of residence as a valid ISO 3166-1 alpha-2 country code e.g. "AU", "NZ", "CA" or an empty string ("") to unset a previously set value. Only applicable, and mandatory if income type is WORKINGHOLIDAYMAKER. 
```

Comment:
We don't do this kind of validation in the SDKs so is the API set up to return an error validation message we can surface?

Reply:
Yep, the API will return a variety of errors on any and all invalid values / combinations as documented.